### ISSUE TITLE
Naming issue "public administration"

### ISSUE THREAD

**Issue by libremente** [italia/publiccode.yml#36]
# Proposal for discussion
The term `public administration`, now used throughout the schema, should be reviewed.
Proposals:
* `public bodies`
* `public authorities`
* others...

@alranel

**Comment by bvhme** [comment]
This is quite a difficult one, I think Public Administration is used by the EU as the non-political part of government and is perhaps the most defined option available, but I am very interested in what viable alternatives would be :)

**Comment by alranel** [comment]
@bvhme you're right, _"Public Administration"_ is indeed correct when talking about the public sector in general. But it's not correct when used for naming a single agency, office or entity. In this repository there are a couple instances of the plural form _"public administrations"_ which should be probably replaced with _"public entities"_. What do you think?
